db/migrate/20181026182144_add_tasks_table.rb

```diff
@@ -0,0 +1,13 @@
+class AddTasksTable < ActiveRecord::Migration[5.1]
+  def change
+    create_table :tasks, :id => :bigserial do |t|
+      t.string :name
+      t.string :status
+      t.string :state
+      t.string :message
```

Comment:
I assume this will be custom messaging to go along with the progress?

Reply:
Right this might be something that we would show as a summary on the UI